src/test/java/ladder/domain/PointTest.java

```diff
@@ -0,0 +1,75 @@
+package ladder.domain;
+
+import ladder.exception.PointException;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static ladder.domain.Point.NOT_ALLOWED_CONSECUTIVE_LINE_MESSAGE;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+
+public class PointTest {
+    @Test
+    @DisplayName("첫번째 포인트를 생성한다")
+    void first() {
+        Point actual = Point.first(true);
+        Point expected = Point.first(true);
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("중간 포인트를 생성한다")
+    void next() {
+        Point actual = Point
+                .first(true).next(false);
+        Point expected = Point.first(false)
+                .next(true).next(false);
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("마지막 포인트를 생성한다")
+    void last() {
+        Point actual = Point.first(false)
+                .next(true).last();
+        Point expected = Point.first(true).next(false)
+                .next(true).last();
+
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @Test
+    @DisplayName("포인트 생성 오류: 라인이 나란히 있을 경우 오류가 발생한다")
+    void create_나란히_line_있을_경우_에러() {
+        assertThatThrownBy(() -> Point.first(true).next(true))
+                .isInstanceOf(PointException.class)
```

Comment:
내부 구현이 변경되어 first 메서드에서 예외가 발생해도 이 테스트는 통과하게 됩니다.
테스트 대상이 되는 메서드만 인자로 전달하면 어떨까요?
```suggestion
        assertThatThrownBy(() -> point.next(true))
                .isInstanceOf(PointException.class)
```

Reply:
언급하신대로 테스트 타깃은 next 메서드의 오류를 재현하는것인데, first 가 감싸져있었네요

언급하신게  테스트 이것 하나이지만,
다시 생각해보니 이 테스트만 해당되는게 아닌것 같습니다.

수정사항 반영해두겠습니다😅